script/lib/install-apm.js

```diff
@@ -4,12 +4,27 @@ const childProcess = require('child_process');
 
 const CONFIG = require('../config');
 
-module.exports = function(ci) {
+module.exports = function(ci, promise = false) {
```

Comment:
I can't just change the function, because this (and `runApmInstall` below) is used elsewhere in places I don't want to touch in this PR (installing test dependencies).